source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -0,0 +1,437 @@
+		return [
+			AddonActionVM(
+				# Translators: Label for an action that installs the selected addon
+				displayName=pgettext("addonStore", "&Install"),
+				actionHandler=self.getAddon,
+				validCheck=lambda aVM: aVM.status == AvailableAddonStatus.AVAILABLE,
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that installs the selected addon
+				displayName=pgettext("addonStore", "&Install (override incompatibility)"),
+				actionHandler=self.installOverrideIncompatibilityForAddon,
+				validCheck=lambda aVM: (
+					aVM.status == AvailableAddonStatus.INCOMPATIBLE
+					and aVM.model.canOverrideCompatibility
+				),
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that updates the selected addon
+				displayName=pgettext("addonStore", "&Update"),
+				actionHandler=self.getAddon,
+				validCheck=lambda aVM: aVM.status == AvailableAddonStatus.UPDATE,
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that replaces the selected addon with
+				# an add-on store version.
+				displayName=pgettext("addonStore", "Re&place"),
+				actionHandler=self.replaceAddon,
+				validCheck=lambda aVM: aVM.status == AvailableAddonStatus.REPLACE_SIDE_LOAD,
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that disables the selected addon
+				displayName=pgettext("addonStore", "&Disable"),
+				actionHandler=self.disableAddon,
+				validCheck=lambda aVM: aVM.model.isInstalled and aVM.status not in (
+					AvailableAddonStatus.DISABLED,
+					AvailableAddonStatus.PENDING_DISABLE,
+					AvailableAddonStatus.INCOMPATIBLE_DISABLED,
+					AvailableAddonStatus.PENDING_INCOMPATIBLE_DISABLED,
+					AvailableAddonStatus.PENDING_REMOVE,
+				),
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that enables the selected addon
+				displayName=pgettext("addonStore", "&Enable"),
+				actionHandler=self.enableAddon,
+				validCheck=lambda aVM: (
+					aVM.status == AvailableAddonStatus.DISABLED
+					or aVM.status == AvailableAddonStatus.PENDING_DISABLE
+				),
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that enables the selected addon
+				displayName=pgettext("addonStore", "&Enable (override incompatibility)"),
+				actionHandler=self.enableOverrideIncompatibilityForAddon,
+				validCheck=lambda aVM: (
+					aVM.status in (
+						AvailableAddonStatus.INCOMPATIBLE_DISABLED,
+						AvailableAddonStatus.PENDING_INCOMPATIBLE_DISABLED,
+					)
+					and aVM.model.canOverrideCompatibility
+				),
+				listItemVM=selectedListItem
+			),
+			AddonActionVM(
+				# Translators: Label for an action that removes the selected addon
+				displayName=pgettext("addonStore", "&Remove"),
+				actionHandler=self.removeAddon,
+				validCheck=lambda aVM: (
+					aVM.model.isInstalled

```

Comment:
```suggestion
					aVM.model.isInstalled
					and self._filteredStatusKey in (
						# Showing remove in the updatable add-ons view is useless
						_StatusFilterKey.INSTALLED
					)
					and aVM.status != AvailableAddonStatus.PENDING_REMOVE
				),
```

Reply:
this should also include `_StatusFilterKey.INSTALLED` as well